index.rst

```diff
@@ -10,20 +10,849 @@
+
+  4. (`SITCOM-180`_) Provide a list of required non-scalar metrics are required and cannot be computed with analysis_tools.
+     Suggest a mechanism (work flow) to perform the measurement, document the finding, evaluate any trend (if applicable), then present it to the stakeholders.
+
+
+.. related to https://confluence.lsstcorp.org/display/LSSTCOM/Displaying+scalar+fields+as+a+function+of+other+parameters
+
+This charge question covers the issue of calculating and displaying the trending of scalar fields.
+Scalar fields are single value metrics, but calculated per spatial element, as described in `Deliverable 3`_.
+This charge question deals with adding a third dimension to the scalar field, then calculating and displaying this data to the user.
+For example, this could be displaying the PSF width for each detector as a function of elevation, or sky transparency as a function of time.
+As discussed above, both of these examples deal with aggregated (binned) data.
+
+Currently, `analysis_tools`_ computes a bundle of single-valued (scalar) metrics on individual visits.
+With small modifications, the package could persist arrays of metric values (e.g., per detector or finer granularity) that could be aggregated and visualized in flexible ways by downstream tooling.
+The package already produces and persists static plots for displaying scalar fields in focal plane coordinates.
+
+After analyzing the use-cases, including hypotheticals not detailed in the report, it was decided that there is not a use-case where we are unable represent a scalar field with respect to a third axis (e.g. time, elevation etc) as a single valued metric (e.g. a mean, or standard deviation), so long as the desired aggregation is supported.
+Taking the examples discussed above, one would reduce the scalar field to a number of scalar metrics, such as the mean PSF width, or the standard deviation about that mean, as a function of elevation.
+Similarly, the sky transparency could be handled by looking at the standard deviation compared to a 2-d map of a photometric night.
+Reducing a scalar field to a scalar metric creates a more generalizable framework to communicate data, however, it comes a the expense of removing information.
+
+The most concerning issue with representing a field as a single metric is that it can hide underlying systematics, such as having only one side of the field having an effect, which is not noticed when looking only at a single number representing the entire field.
+For this reason, and for the more general reason of needing the ability to dig into the data when a metric is not within the expected range, it is required to have the ability to view and reproduce the data that went into calculating the analysis_tools metric.
+`FAFF-REQ-0059`_ has been created to capture the functionality of writing to disk both the calculated metric, and the object that was used to determine it.
+This capability is now realized by the refactored analysis_tools design.
```

Comment:
Do you mean "implemented" when you say "realized"?

Reply:
Yes and no... it means the design incorporates it, but it might not be implemented.
This language is used in risk analysis a lot.
A risk is "realized" when you say, "ya, this is in the process of happening"  even though it hasn't happened yet.